tests/test_jetpack.py

```diff
@@ -328,20 +300,23 @@ def test_mismatched_db_hash():
 
     err = _do_test(MockXPI({"bootstrap.js": bootstrap,
                             "resources/bootstrap.js": bootstrap,
-                            "harness-options.json":
-                                json.dumps(harnessoptions)}))
-    print err.print_summary(verbose=True)
-    assert not err.failed()
+                            "harness-options.json": harnessoptions}))
+    assert err.failed()
```

Comment:
Yes. There was no warning for this in the old code. The new code reports files which are expected to match a known hash but don't.